src/model/model.ts

```diff
@@ -230,6 +247,23 @@ export class Model {
     return false;
   }
 
+  // removePolicies removes policy rules from the model.
+  public removePolicies(sec: string, ptype: string, rules: string[][]): boolean {
+    for (const rule of rules) {
+      if (this.hasPolicy(sec, ptype, rule)) {
+        const ast = this.model.get(sec)?.get(ptype);
```

Comment:
The implementation is correct but we need to avoid getting `ast` many times. 
Move it to the outside of `for`.